google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java

```diff
@@ -40,7 +45,9 @@
   private JTable bucketContentTable;
   private GcsBlobTableModel tableModel;
 
-  GcsBucketContentEditorPanel(@NotNull Bucket bucket) {
+  private static final Color MEDIUM_GRAY = new Color(96, 96, 96);
+
+      GcsBucketContentEditorPanel(@NotNull Bucket bucket) {
```

Comment:
nit: Looks like the indentation here got messed up

Reply:
fixed